src/FixedDataTableWidthHelper.js

```diff
@@ -22,6 +22,33 @@ function getTotalWidth(/*array*/ columns) /*number*/ {
   return totalWidth;
 }
 
+function getMaxVisibleColumns(/*array*/ columns, /*number*/ width) /*number*/ {
+  var maxColumns = 0;
+  var currentWidth = 0;
+
+  var scrollableColumns = columns.filter(function(column) {
+    var fixed = column.props.fixed;
+    if (fixed) {
+      width -= column.props.width;
```

Comment:
This is a side-effect. Can you either break this out into a different loop, or put a comment here about how we do extra work here to subtract the fixed column widths from the total width?